db/schema.rb

```diff
@@ -27,14 +27,15 @@
   add_index "procurement_areas", ["locations"], name: "index_procurement_areas_on_locations", using: :gin
   add_index "procurement_areas", ["memberships"], name: "index_procurement_areas_on_memberships", using: :gin
 
-  create_table "users", force: :cascade do |t|
-    t.integer  "defence_request_uid"
-    t.integer  "defence_request_access_token"
-    t.string   "email"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+  create_table "shifts", force: :cascade do |t|
+    t.string   "name"
+    t.string   "location_uid",  null: false
+    t.time     "starting_time"
+    t.time     "ending_time"
```

Comment:
That is probably a good call, my only concern is that, without further information as to what we might be calculating, we might not store it in an efficient way.